pkgs/tools/security/tessen/default.nix

```diff
@@ -32,6 +33,8 @@ stdenvNoCC.mkDerivation rec {
   '';
 
   postInstall = ''
+    scdoc < man/tessen.1.scd > man/tessen.1
+    scdoc < man/tessen.5.scd > man/tessen.5
     installManPage man/*
```

Comment:
For what is the file ``share/man/manscd/tessen.1.scd.gz`` and is that expected to be in $out?

Reply:
Good to know. I wasn't sure if they are something I just didn't know about.